From 4dd701916186803172a9f35e7e982a953613ad55 Mon Sep 17 00:00:00 2001
From: Ansuel Smith <ansuelsmth@gmail.com>
Date: Mon, 11 Apr 2022 21:32:41 +0200
Subject: [PATCH 5/9] nss-drv: use standard skb_skip_tc_classify instead of
 custom api

Use skb_skip_tc_classify to skip classify for packet handled by nss
instead of custom api.

Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>
---
 nss_core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nss_core.c b/nss_core.c
index f9e6014..6ab8038 100644
--- a/nss_core.c
+++ b/nss_core.c
@@ -1075,7 +1075,7 @@ static inline void nss_core_set_skb_classify(struct sk_buff *nbuf)
 #if (LINUX_VERSION_CODE < KERNEL_VERSION(5, 4, 0))
 		nbuf->tc_verd = SET_TC_NCLS_NSS(nbuf->tc_verd);
 #else
-		skb_set_tc_classify_offload(nbuf);
+		skb_skip_tc_classify(nbuf);
 #endif
 #endif
 }
--
2.34.1
